sympy/geometry/tests/test_ellipse.py

```diff
@@ -9,6 +9,18 @@
 from sympy import integrate
 from sympy.functions.special.elliptic_integrals import elliptic_e
 
+def test_object_from_equation():
+    from sympy.abc import x, y
+    assert Circle.object_from_equation(x ** 2 + y ** 2 + 3 * x + 4 * y - 8), Circle(Point2D(-3 / 2, -2), sqrt(57) / 2)
```

Comment:
@sidhantnagpal  No, I didn't. 
When I tried using `==` it resulted in errors.
Then I saw a few similar tests in `test_line.py` which resulted in `objects` and they were written in a similar format ( i.e. using `,` instead of `==` ), so I used `,` instead.

Reply:
Did you refer to your own tests?
By the way, the errors might be due to the fact that you are using floats and not Rationals.
For the above case it should be `Circle(Point2D(-S(3)/2, -2), sqrt(57)/2)`.

Reply:
This is not correct. I think `==` should be used here.
```
assert Line3D.angle_between(Line3D(Point3D(0, 0, 0), Point3D(1, 1, 1)),
                                Line3D(Point3D(0, 0, 0), Point3D(5, 0, 0))) == acos(sqrt(3) / 3)
```

Reply:
@sidhantnagpal  Okay, Looks like you were correct. There aren't any errors coming up now in using `==`. 
I've made another commit. Please check.